diff --git a/sesman/env.c b/sesman/env.c
index d8bb1e9..3d8d413 100644
--- a/sesman/env.c
+++ b/sesman/env.c
@@ -151,11 +151,8 @@ env_set_user(char *username, char **passwd_file, int display,
                 {
                     /* if no auth_file_path is set, then we go for
                      $HOME/.vnc/sesman_username_passwd */
-                    if (g_mkdir(".vnc") < 0)
-                    {
-                        log_message(LOG_LEVEL_ERROR,
-                                    "env_set_user: error creating .vnc dir");
-                    }
+                    /* ignore any errors from mkdir for now, since .vnc might already exist */
+                    g_mkdir(".vnc");
 
                     len = g_snprintf(NULL, 0, "%s/.vnc/sesman_%s_passwd", pw_dir, username);
 
